morph_tool/utils.py

```diff
@@ -33,24 +33,9 @@ def neurondb_dataframe(filename: Path) -> pd.DataFrame:
         columns = ['name', 'layer', 'mtype']
         df = pd.read_csv(filename, sep=r'\s+', names=columns, usecols=range(len(columns)))
         df.layer = df.layer.astype('str')
-    elif filename.suffix.lower() == '.xml':
-        with filename.open() as fd:
-            neuronDB = xmltodict.parse(fd.read())
-
-        rows = list()
-        for morph in neuronDB["neurondb"]["listing"]["morphology"]:
-            assert morph["repair"]["use_axon"] in {'true', 'false', 'True', 'False', None}
-            rows.append(
-                (morph["name"],
-                 morph["layer"],
-                 morph["mtype"] + (":" + morph["msubtype"] if morph["msubtype"] else ""),
-                 # According to Eilif, an empty use_axon (corresponding to a null in the database)
-                 # means that the axon is supposed to be used
-                 (morph["repair"]["use_axon"] in {'true', 'True', None})))
-
-        df = pd.DataFrame(data=rows, columns=['name', 'layer', 'mtype', 'use_axon'])
+        return df
```

Comment:
so this only reads .dat now? maybe update the docstring?

Reply:
No, it also reads XML, see (new) line 38. Once again, GH diff formatting is weird.

Reply:
ah indeed ;)